packages/core/src/components/Tooltip/Tooltip.tsx

```diff
@@ -196,7 +200,10 @@ export default class Tooltip extends PureComponent<TooltipProps> {
     }
 
     return (
-      <div style={style} className={cx(styles.tooltip, getStyle(styles, camelCase(theme)), className)}>
+      <div
+        style={{ ...style, "--tooltip-max-width": `${maxWidth}px` } as CSSProperties}
```

Comment:
If maxWidth is not sent wont it cause it to be `undefinedpx`?

Reply:
oops, good catch! fixing